app/javascript/chat/RequestManager/ChannelRequestSection.jsx

```diff
@@ -20,6 +20,7 @@ export const ChannelRequestSection = ({
         channelRequests.map((channelPendingRequest) => {
           return (
             <RequestListItem
+              key={`${channelPendingRequest.channel_name}-${channelPendingRequest.membership_id}`}
```

Comment:
Refactor (blocking): `channel_name` is adequate for the key, as the context of this rendered component is for a specific user.